jobs/kola-openstack.Jenkinsfile

```
@@ -43,97 +41,105 @@ properties([
     durabilityHint('PERFORMANCE_OPTIMIZED')
 ])
 
-currentBuild.description = "[${params.STREAM}][${params.ARCH}] - ${params.VERSION}"
+lock(resource: "kola-openstack-${params.ARCH}") {
 
-def s3_stream_dir = params.S3_STREAM_DIR
-if (s3_stream_dir == "") {
-    s3_stream_dir = "fcos-builds/prod/streams/${params.STREAM}"
-}
+    currentBuild.description = "[${params.STREAM}][${params.ARCH}] - ${params.VERSION}"
+    region = "ams1"
```

Might need `def` in the front like is done for other variables.

```suggestion
    def region = "ams1"
```

---

Might also be good to have a comment above this block that explains **why** we want to send different architectures to different regions. Basically that we don't have enough quota in a single region to do multiple runs at the same time but we want to speed up our testing so we'll target two regions (i.e. we can run 2 runs at the same time (an `aarch64` and `x86_64`)).

---

Maybe that comment should go above the `lock()` block.